test/unit/apicast/sandbox_proxy_test.rb

```diff
@@ -8,6 +8,6 @@ def test_proxy_host
     service.proxy.apicast_configuration_driven = false
     service.proxy.sandbox_endpoint = nil
 
-    assert_equal "#{service.system_name}-#{service.account_id}.staging.apicast.io", proxy.proxy_host
+    assert_equal "#{service.system_name}-#{service.account_id}.staging.apicast.dev", proxy.proxy_host
```

Comment:
Also I can't understand why this change is needed.

Reply:
with the removal of apicastv1 class ".io" endpoint doesn't exist

Reply:
Would you tell me how that works? I don't understand why it stopped existing.